CMakeLists.txt

```diff
@@ -1,677 +1,87 @@
-project(OpenColorIO)
-set(OCIO_VERSION_MAJOR 1)
-set(OCIO_VERSION_MINOR 1)
-set(OCIO_VERSION_PATCH 0)
+cmake_minimum_required(VERSION 3.1)
+project(OpenColorIO VERSION 1.2.0)
+
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
```

Comment:
Based on [this page](https://crascit.com/2015/03/28/enabling-cxx11-in-cmake/) it seems that `set(CMAKE_CXX_EXTENSIONS OFF)` must be added to avoid platform specific C++11 extensions.